api-report/telemetry-utils.api.md

```diff
@@ -179,6 +179,16 @@ export function raiseConnectedEvent(logger: ITelemetryLogger, emitter: EventEmit
 // @public (undocumented)
 export function safeRaiseEvent(emitter: EventEmitter, logger: ITelemetryLogger, event: string, ...args: any[]): void;
 
+// @public
+export class SamplingLoggerAdapter implements ITelemetryBaseLogger {
```

Comment:
i don't think this change should be here. there is no code change in the telemetry-utils package